src/Resources/Resources.Sdk/Readme.md

```diff
@@ -36,7 +36,7 @@ These settings apply only when `--tag=package-deploymentscripts-2020-10` is spec
 
 ```yaml $(tag) == 'package-deploymentscripts-2020-10'
 input-file:
-- https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json
+- https://github.com/Azure/azure-rest-api-specs/blob/20a98a8408336d47697562f4cdad451f1d0216bf/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json
```

Comment:
Could you use just one commit id for all swaggers that will make it easy for us to main? Besides, you may define a **commit** variable as the following example, and next time you update commit id you will just need to update the commit variable.
```
commit: 378977e71101fdf224414b86e9f7d8c776cd07ec
input-file: 
  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json
```

Reply:
Made the change